go/service/kbfs_favorites_handler.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// Handlers for KBFS-favorites-related gregor messages
+
+package service
+
+import (
+	"fmt"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/gregor"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/gregor1"
+)
+
+const kbfsFavoritesHandlerName = "kbfsFavoritesHandler"
+
+type kbfsFavoritesHandler struct {
+	libkb.Contextified
+}
+
+var _ libkb.GregorInBandMessageHandler = (*ekHandler)(nil)
```

Comment:
did you mean to use *kbfsFavoritesHandler here?  instead of ekHandler

Reply:
yes I did.. copy-paste mishap.